src/gso.json

```diff
@@ -465,8 +605,5 @@
       "name": "InvalidLockupEnd",
       "msg": "Lockup end cannot be before subscription period end"
     }
-  ],
-  "metadata": {
-    "address": "DuALd6fooWzVDkaTsQzDAxPGYCnLrnWamdNNTNxicdX8"
```

Comment:
why remove?

Reply:
it doesnt matter. gets put in there or not depending on your anchor version when building. we dont use it